### DISCUSSION TITLE
regexp not work in site list？

### DISCUSSION THREAD

**Discussion by Asukaaaaaa** [darkreader/darkreader#11933] [Q&A]
![image](https://github.com/darkreader/darkreader/assets/24503369/95b90462-7be7-48de-819c-d61460596833)
Now only the second one makes dark reader work on bilibili.com. But the first one worked fine before.

**Comment by alexanderby** [comment]
Hi, if you want a regexp, it should start with `/` and end with `/`, like `/(bilibili).com/`
